homeassistant/components/synology/switch.py

```diff
@@ -0,0 +1,64 @@
+"""Support for Synology Surveillance Station Cameras."""
```

Comment:
Please do the refactor first and then when that's merged follow up with adding the switch in another PR.